packages/frontend/src/mixpanel/index.js

```diff
@@ -39,7 +39,9 @@ let Mixpanel = {
     register: () => {}
 };
 
-if (BROWSER_MIXPANEL_TOKEN) {
+const shouldEnableTracking = false; // BROWSER_MIXPANEL_TOKEN
+
+if (shouldEnableTracking) {
     mixpanel.init(BROWSER_MIXPANEL_TOKEN);
```

Comment:
What happens here? Do we send data to MixPanel?

Reply:
Yep we send data to MixPanel if there's a token coming from the config

Reply:
@Patrick1904 Are your trying to remove mixpanel tracking for wallet.near.org domains?